test/unit/programs.js

```diff
@@ -0,0 +1,495 @@
+process.env.NODE_ENV = 'development';
+const programs = require('../../routes/programs');
+const seed = require('../../lib/utils').seed;
+const chai = require('chai');
+const assert = chai.assert;
+
+
+const program1 = {program_id: 1, site_id: 1, program_name: 'test program'};
+const program2 = {program_id: 2, site_id: 2, program_name: 'test program 2'};
+const program3 = {program_id: 3, site_id: 10, program_name: 'test program 3'};
+const program4 = {program_id: 4, site_id: 11, program_name: 'test program 4'};
+var allPrograms = [program1, program2, program3, program4];
+
+var programsDeleted = [program2, program3, program4];
+const newProgram = {program_id: 5, site_id: 5, program_name: 'new test program'};
+var programsNew = [program1, program2, program3, program4, newProgram];
+const updatedProgram = {program_id: 1, site_id: 1, program_name: 'updated program'};
+
+describe('GET', function(done) {
```

Comment:
Take a look at the [students tests](https://github.com/CS4500-AmericaSCORES/america-scores-backend/blob/master/test/unit/students.js). We should unify the way we describe our tests, and I think the way @jacquelineali described her tests is really great for a quick idea of what's going right/wrong.